packages/styles/src/createRenderer.ts

```diff
@@ -0,0 +1,32 @@
+import { createRenderer as createFelaRenderer, TPlugin } from 'fela';
+import felaPluginUnit from 'fela-plugin-unit';
+import felaPluginEmbedded from 'fela-plugin-embedded';
+import felaPluginMultipleSelectors from 'fela-plugin-multiple-selectors';
+import felaSortMediaQueryMobileFirst from 'fela-sort-media-query-mobile-first';
+import felaPluginPlaceholderPrefixer from 'fela-plugin-placeholder-prefixer';
+import felaEnforceLonghands from 'fela-enforce-longhands';
+import felaMonolithic from 'fela-monolithic';
+
+import { makeResponsiveValuePlugin } from './plugins/responsiveValues';
+import { makeTypedSelectorsPlugin } from './plugins/typedSelectors';
+
+export const createRenderer = () =>
+    createFelaRenderer({
+        devMode: process.env.NODE_ENV === 'development',
+        enhancers: [
+            felaEnforceLonghands(),
+            felaMonolithic({ prettySelectors: true }),
```

Comment:
this plugin is not meant to be used in production? https://github.com/robinweser/fela/blob/master/packages/fela-monolithic/README.md?plain=1#L5

Reply:
I will disable it for production.